.github/workflows/build-images.yml

```diff
@@ -146,6 +149,9 @@ jobs:
         timeout: '20m0s'
         format: 'table'
         output: 'trivy-results-vertica-image.out'
+      env:
+        TRIVY_DB_REPOSITORY: ghcr.io/aquasecurity/trivy-db,public.ecr.aws/aquasecurity/trivy-db
```

Comment:
Could we define the env vars at job level then we can easily change the repos in future?

Reply:
Sure, updated in https://github.com/vertica/vertica-kubernetes/pull/957/commits/cbdbd45965fc99d402b47c73742878051c3448ea